src/components/editNotePage/EditNotePage.tsx

```diff
@@ -0,0 +1,73 @@
+import React, { Component, Props } from 'react';
+// tslint:disable-next-line:no-implicit-dependencies
+import { RouteComponentProps } from 'react-router';
```

Comment:
You don't need to import it from `react-router` and then try to hush the `no-implicit-dependencies` rule.
You may just import it from `react-router-dom`:

```js
import { Link, RouteComponentProps } from 'react-router-dom';
```

Because `react-router-dom` re-exports things from `react-router`:
```js
// react-router-dom/index.d.ts
export {
    generatePath,
    Prompt,
    MemoryRouter,
    RedirectProps,
    Redirect,
    RouteComponentProps,
    RouteProps,
    Route,
    Router,
    StaticRouter,
    Switch,
    match,
    matchPath,
    withRouter,
    RouterChildContext
} from 'react-router';
```

Reply:
thanks! fixed 👍